src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -245,8 +245,9 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
             builder.ConfigureServices((services, serviceProvider) =>
             {
                 var settings = serviceProvider.GetRequiredService<ShellSettings>();
+                var environment = serviceProvider.GetRequiredService<IHostEnvironment>();
 
-                var cookieName = "orchantiforgery_" + settings.Name;
+                var cookieName = "orchantiforgery_" + settings.Name + environment.ContentRootPath;
```

Comment:
Doesn't it need to be url encoded?

Reply:
Yes, but as i saw it is done automatically by aspnetcore

But just quickly done, maybe better to do it explicitly, will do